### PR TITLE
RFC: The WatermarkFilter and StreamSort operator

### PR DESCRIPTION
**Pull request by TennyZhuang**
Signed-off-by: TennyZhuang <zty0826@gmail.com>

### CODE REVIEW

**Comment by TennyZhuang** [comment]
image
image

File: rfcs/0002-watermark-filter.md [hunk 1]
```diff
@@ -235,6 +235,69 @@ WINDOW w AS (
 )
 ```
 
+### watermark_strategy_expression
+
+> the section is updated on 31 Oct
```
Comment (TennyZhuang) [inline rfcs/0002-watermark-filter.md:240]: I guess it's unnecessary :(

**Comment by liurenjie1024** [comment]
Here is databrick sql's syntax: https://docs.databricks.com/sql/language-manual/sql-ref-window-functions.html It's much more concise than pg.

**Review by fuyufjh — APPROVED** [review]
Let's merge this.

**Pull request merged**